services/minespace-web/src/components/Forms/projects/majorMineApplication/MajorMineApplicationReviewSubmit.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { CALLOUT_SEVERITY } from "@common/constants/strings";
+import { Checkbox, Row, Col, Typography, Descriptions, Card, Input } from "antd";
+import PropTypes from "prop-types";
+import * as Strings from "@common/constants/strings";
+import {
+  removeDocumentFromMajorMineApplication,
+  fetchProjectById,
+} from "@common/actionCreators/projectActionCreator";
+import Callout from "@/components/common/Callout";
+import DocumentTable from "@/components/common/DocumentTable";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  project: CustomPropTypes.project.isRequired,
+  confirmedSubmission: PropTypes.bool.isRequired,
+  removeDocumentFromMajorMineApplication: PropTypes.func.isRequired,
+  fetchProjectById: PropTypes.func.isRequired,
+  setConfirmedSubmission: PropTypes.func.isRequired,
+};
+
+const inputStyle = { width: "100%" };
+
+export const MajorMineApplicationReviewSubmit = ({
+  project,
+  removeDocumentFromMajorMineApplication: removeDocument,
+  fetchProjectById: fetchProject,
+  confirmedSubmission,
```

Comment:
is there a reason you didnt just hand these into the component as props?

Reply:
They are props, I am just destructuring props at the component level. No specific reason, just how it came out.